pydantic_core/_pydantic_core.pyi

```diff
@@ -125,6 +130,8 @@ class MultiHostUrl:
 class SchemaError(Exception):
     pass
 
+# Note: ErrorDetails is copied into __init__.py.
```

Comment:
can't we replace this with `from . import ErrorDetails`?

Should work since this is for typing analysis only by definition.

Reply:
Hm. Good question. I'll give that a shot (probably tonight or tomorrow) and report back.